shared/wallets/transaction/index.js

```diff
@@ -105,6 +106,7 @@ const Detail = (props: DetailProps) => {
   const textTypeSemibold = props.large ? 'BodySemibold' : 'BodySmallSemibold'
   const textTypeSemiboldItalic = props.large ? 'BodySemiboldItalic' : 'BodySmallSemiboldItalic'
   const textTypeExtrabold = props.large ? 'BodyExtrabold' : 'BodySmallExtrabold'
+  const textSentenceEnd = props.detailView && props.pending ? '\u2026' : '.'
```

Comment:
:sheep: a comment saying `\u2026` is an ellipsis would be nice

Reply:
does that mean you are asking sheepishly?

Reply:
Lol, it's a suggestion but not a deal breaker

Reply:
just wondering what sheep means